src/tree/SubscriptionTreeItem.ts

```diff
@@ -97,7 +98,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
         const promptSteps: AzureWizardPromptStep<IAppServiceWizardContext>[] = [];
         const executeSteps: AzureWizardExecuteStep<IAppServiceWizardContext>[] = [];
-        promptSteps.push(new SiteNameStep());
+        promptSteps.push(new SiteNameStep({ parameterDisplayTitle: "Function App Name", parameterDisplayDescription: "The name of the new function app." }));
```

Comment:
I know that there are cases where `SiteNameStep` is used for Functions or for Web Apps, but I still think it might be better to define these property in the step class itself rather than passing them as options?

To distinguish between the two, I think we can pass an option for web app or function app, but having to add the parameter information each time we create a new step seems like it'd be cumbersome especially for other steps (assuming that we follow that pattern for all steps)

Reply:
Ok. This will require changes in azuretools.

Reply:
https://github.com/microsoft/vscode-azuretools/pull/1694

Reply:
Updated.